benchmark/single-source/SetTests.swift

```diff
@@ -94,46 +99,56 @@ public let SetTests = [
     setUpFunction: { blackHole([setP, setQ]) }),
 
   BenchmarkInfo(
-    name: "Set.Empty.IsDisjointInt0",
-    runFunction: { n in run_SetIsDisjointInt(setE, setAB, true, 50 * n) },
+    name: "Set.isDisjoint.Empty.Int0",
+    runFunction: { n in run_SetIsDisjointInt(setE, setAB, true, 5000 * n) },
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setE, setAB]) }),
   BenchmarkInfo(
-    name: "Set.Empty.IsDisjointBox0",
-    runFunction: { n in run_SetIsDisjointBox(setOE, setOAB, true, 50 * n) },
+    name: "Set.isDisjoint.Int0.Empty",
+    runFunction: { n in run_SetIsDisjointInt(setAB, setE, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, setE]) }),
+  BenchmarkInfo(
+    name: "Set.isDisjoint.Empty.Box0",
+    runFunction: { n in run_SetIsDisjointBox(setOE, setOAB, true, 5000 * n) },
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setOE, setOAB]) }),
   BenchmarkInfo(
-    name: "SetIsDisjointInt0",
+    name: "Set.isDisjoint.Box0.Empty",
+    runFunction: { n in run_SetIsDisjointBox(setOAB, setOE, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, setOE]) }),
+  BenchmarkInfo(
+    name: "Set.isDisjoint.Int0",
     runFunction: { n in run_SetIsDisjointInt(setAB, setCD, true, 50 * n) },
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setAB, setCD]) }),
   BenchmarkInfo(
-    name: "SetIsDisjointBox0",
+    name: "Set.isDisjoint.Box0",
     runFunction: { n in run_SetIsDisjointBox(setOAB, setOCD, true, 50 * n) },
```

Comment:
This is weird. `Set.isDisjoint.Int0` should be comparable in performance to `Set.isDisjoint.Int0`, but judging by the factor here and benchmark results, it's apparently 300 times slower.

Reply:
Added |   |   |  
-- | -- | -- | --
Set.isDisjoint.Int0 | 1338 | 1400 | 1359
Set.isDisjoint.Box0 | 3153 | 3268 | 3206 | —

...both have multiplier 50, so I see only 3x slowdown. Right?

Reply:
Hah, you're right -- I matched it up with the Empty case by accident. Another +1 for the new names.

3x is fine!